zrml/prediction-markets/src/lib.rs

```diff
@@ -2150,6 +2148,12 @@ mod pallet {
             Ok(())
         }
 
+        /// The reserve ID of the court pallet.
```

Comment:
```suggestion
        /// The reserve ID of the prediction-markets pallet.

Reply:
Mitigated in [587e90e](https://github.com/zeitgeistpm/zeitgeist/pull/749/commits/587e90e16f55e16ff7ba7b296f63da2776ba4e11)